examples/inference/gpt/gpt_dynamic_inference.py

```diff
@@ -117,8 +117,11 @@ def get_model() -> MegatronModule:
     return model
 
 
-def get_inference_context(requests: List[Request], sampling_params: SamplingParams, 
-                          calculate_max_sequence_length_from_requests: bool =True):
+def get_inference_context(
+    requests: List[Request],
+    sampling_params: SamplingParams,
```

Comment:
Can we make this optional? I guess it should only be required if we're setting `calculate_max_sequence_length_from_requests`?